scandium-core/src/main/java/org/eclipse/californium/scandium/DTLSConnector.java

```diff
@@ -785,6 +795,9 @@ private void processAlertRecord(final Record record, final Connection connection
 					errorHandler.onError(alert.getPeer(), alert.getLevel(), alert.getDescription());
```

Comment:
I didn't get it. The one is for catching a failed handshake, the other is to report a send error, which may have other causes (even with successful handshake the send may fail).